### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by krishnanrs**
Updated AWS, GCP and ESX documentation to reflect the current usage
and arguments. Main changes include:
  * Changed GCE to GCP
  * Added brkt-tag argument to all commands
  * Added documentation for the `wrap` command for each of the CSPs

### CODE REVIEW

**Comment by apoorva-jindal** [comment]
ESX changes look good to me.

File: brkt_cli/esx/encrypt_vmdk.py [hunk 1]
```diff
@@ -201,7 +201,9 @@ def encrypt_from_local_ovf(vc_swc, enc_svc_cls, guest_vmdk, crypto_policy,
             return
         # Launch OVF
         log.info("Launching VM from local OVF")
-        ovf_image_name = ovf_image_name + ".ovf"
+        # Normalize image name if require
```
Comment (bburtin) [inline brkt_cli/esx/encrypt_vmdk.py]: Can you put this into a separate commit?

Comment (krishnanrs) [inline brkt_cli/esx/encrypt_vmdk.py]: Done. Please check again.
